eng/targets/Helix.Common.props

```diff
@@ -29,7 +29,7 @@
   <ItemGroup Condition="'$(IsWindowsOnlyTest)' != 'true' AND '$(TargetArchitecture)' == 'x64' AND '$(IsHelixDaily)' == 'true'">
     <HelixAvailableTargetQueue Include="Windows.7.Amd64.Open" Platform="Windows" />
     <HelixAvailableTargetQueue Include="Windows.81.Amd64.Open" Platform="Windows" />
-    <HelixAvailableTargetQueue Include="Ubuntu.1604.Amd64.Open" Platform="Linux" />
+    <HelixAvailableTargetQueue Include="Ubuntu.1804.Amd64.Open" Platform="Linux" />
```

Comment:
?

Reply:
This makes it so we run 1604 on PRs, and only use 1804 on the 2x a day master builds, this is what we were doing before I believe

Reply:
so 1804 wouldn't be used at all on internal builds as those only run required checks now.  helix matrix doesn't run on internal

Reply:
Why change anything in the public runs? The only issue is internal builds.

I'd prefer making this conditional on the `$(SYSTEM_TEAMPROJECT)` value. Perhaps even don't bother testing on Ubuntu in official builds.